src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.MyRider;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.impl.MyRiderImpl;
+import core.basesyntax.service.impl.MyWriterImpl;
+import core.basesyntax.service.impl.ParserImpl;
+import core.basesyntax.service.impl.ReportImpl;
+import core.basesyntax.service.impl.ValidatorImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        MyRider reader = new MyRiderImpl();
+        List<String> lines = reader.readFromFile("./src/main/resources/input.csv");
```

Comment:
can we use static final String readFromFile = ""./src/main/resources/input.csv""  ?